components/layout/ActivityDetails.js

```diff
@@ -45,6 +53,14 @@ export default function ActivityDetails({
               alt="Image is missing"
             />
           )}
+
+        {showHeart ? (
+          <StyledHeartIcon onClick={() => toggleBookmark(id)}>
+            <FaHeart fill={isBookmarked ? "#ff4d4d" : "#fff"} />
```

Comment:
You might want to describe the component name differently to make it more understandable for other readers

Reply:
i changed the name of the container